tests/unittests/helpers/testers.py

```diff
@@ -663,3 +663,13 @@ def remove_ignore_index(target: Tensor, preds: Tensor, ignore_index: Optional[in
         idx = target == ignore_index
         target, preds = deepcopy(target[~idx]), deepcopy(preds[~idx])
     return target, preds
+
+
+def remove_ignore_index_groups(
```

Comment:
I am not sure whether I should create a separate function for this, or just modify the existing ``remove_ignore_index`` to accommodate groups?

Reply:
sounds good to me :)